examples/hooks/unity_table_export.lua

```diff
@@ -0,0 +1,26 @@
+--[[
+ As an exhaustive example, it will first start off with a Delta Lake tables export, then continue to register the table
+ with Unity Catalog
+]]
+
+local aws = require("aws")
+local formats = require("formats")
+local services = require("services")
+local delta_export = require("lakefs/catalogexport/delta_exporter")
+local unity_export = require("lakefs/catalogexport/unity_exporter")
+
+local table_descriptors_path = "_lakefs_tables"
```

Comment:
IMO it's easier on the eye if the example will use `_lakefs_tables` hardcoded in the function calls instead of `table_descriptors_path` variable, Up to you.